openapi-cli/src/main/java/io/ballerina/openapi/cmd/OASContractGenerator.java

```diff
@@ -147,6 +147,11 @@ public void generateOAS3DefinitionsAllService(Path servicePath, Path outPath, St
             while (iterator.hasNext()) {
                 outStream.println("-- " + iterator.next());
             }
+        } else {
+            DiagnosticMessages message = DiagnosticMessages.OAS_CONVERTOR_115;
+            ExceptionDiagnostic error = new ExceptionDiagnostic(message.getCode(),
+                    message.getDescription(), null, message.getSeverity());
```

Comment:
Can't we have an `ExceptionDiagnostic` constructor to directly accept `DiagnosticMessages`, instead of providing the all the fields separately at each time?

Reply:
Reverted the change due to the comment [above](https://github.com/ballerina-platform/openapi-tools/pull/1245#discussion_r1144443406)